mitodl/mit-open
# Update eslint-config and friends

### What are the relevant tickets?
Opened this to check
- https://github.com/mitodl/eslint-config/pull/47

### Description (What does it do?)
Updates our eslint-config and some of its peer dependencies.


### How can this be tested?
1. Reinstall yarn dependencies (`docker compose restart watch` or `yarn install`) and run `yarn lint-check`. It should pass.
2. You could also try triggering an error.


### Notes
⚠️  This is using a git-installed version of eslint-config-mitodl. After that version is published, this PR should be updated.
